rustls-libssl/src/lib.rs

```diff
@@ -357,6 +364,18 @@ pub fn iter_alpn(mut slice: &[u8]) -> impl Iterator<Item = Option<&[u8]>> {
     })
 }
 
+/// Encode rustls's internal representation in the wire format.
+fn encode_alpn<'a>(iter: impl Iterator<Item = &'a [u8]>) -> Vec<u8> {
```

Comment:
nit: Since this is used in `Ssl::invoke_accepted_callbacks()` down below on L810 I think this helper should be defined below the `impl Ssl` block.